app.js

```diff
@@ -63,7 +63,16 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
 // Open the thing database.
-db.open();
+var dbName = './db.sqlite3';
+if (process.env.NODE_ENV === 'test') {
+  // Use a different database for testing.
+  dbName = './test-db.sqlite3';
+  // Always start out with no database (for testing)
+  if (fs.existsSync(dbName)) {
+    fs.unlinkSync(dbName);
+  }
+}
```

Comment:
I think that its possible to do this in the test.js file. I just need to do it before the require('../app') line.